src/dali/abstract_ccxt_input_plugin.py

```diff
@@ -0,0 +1,628 @@
+        out_transactions: List[OutTransaction],
+        intra_transactions: List[IntraTransaction],
+    ) -> None:
+        raise NotImplementedError("Abstract method")
+
+    def _process_trades(
+        self,
+        in_transactions: List[InTransaction],
+        out_transactions: List[OutTransaction],
+    ) -> None:
+
+        processing_result_list: List[Optional[ProcessOperationResult]] = []
+        pagination_detail_set: AbstractPaginationDetailSet = self._get_process_trades_pagination_detail_set()
+        pagination_detail_iterator: AbstractPaginationDetailsIterator = iter(pagination_detail_set)
+        try:
+            while True:
+                pagination_details: PaginationDetails = next(pagination_detail_iterator)
+                trades: Optional[List[Dict[str, Union[str, float]]]] = self._client.fetch_my_trades(
+                    symbol=pagination_details.symbol,
+                    since=pagination_details.since,
+                    limit=pagination_details.limit,
+                    params=pagination_details.params,
+                )
+                #   {
+                #       'info':         { ... },                    // the original decoded JSON as is
+                #       'id':           '12345-67890:09876/54321',  // string trade id
+                #       'timestamp':    1502962946216,              // Unix timestamp in milliseconds
+                #       'datetime':     '2017-08-17 12:42:48.000',  // ISO8601 datetime with milliseconds
+                #       'symbol':       'ETH/BTC',                  // symbol
+                #       'order':        '12345-67890:09876/54321',  // string order id or undefined/None/null
+                #       'type':         'limit',                    // order type, 'market', 'limit' or undefined/None/null
+                #       'side':         'buy',                      // direction of the trade, 'buy' or 'sell'
+                #       'takerOrMaker': 'taker',                    // string, 'taker' or 'maker'
+                #       'price':        0.06917684,                 // float price in quote currency
+                #       'amount':       1.5,                        // amount of base currency
+                #       'cost':         0.10376526,                 // total cost, `price * amount`,
+                #       'fee':          {                           // provided by exchange or calculated by ccxt
+                #           'cost':  0.0015,                        // float
+                #           'currency': 'ETH',                      // usually base currency for buys, quote currency for sells
+                #           'rate': 0.002,                          // the fee rate (if available)
+                #       },
+                #   }
+
+                # * The work on ``'fee'`` info is still in progress, fee info may be missing partially or entirely, depending on the exchange capabilities.
```

Comment:
I think it may be misleading to paste a direct quote from their docs, because their software may change and then our comment would be out of sync. How about a link to their doc or nothing at all?